app/helpers/home_helper.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module HomeHelper
+  def home_page_tiles
```

Comment:
Very nitpicky but I think this should be fine to just assign to a `@tiles` var in the controller. Otherwise this looks good! 👍